livelike/tests/test_attribution.py

```diff
@@ -0,0 +1,466 @@
+import os
+
+import numpy
+import pandas
+
+from livelike import _attribution
+
+eval_data_person = pandas.read_parquet(
```

Comment:
[Pre-commit is failing](https://results.pre-commit.ci/run/github/951924054/1746721171.Kee-9ujNQZCo2GDAWhIwCA) -- line too long -- needs formatting

```
livelike/tests/test_attribution.py:9:89: E501 Line too long (90 > 88)
   |
 8 | eval_data_person = pandas.read_parquet(
 9 |     os.path.join("livelike", "tests", "attribution_eval_data", "eval_data_person.parquet")
   |                                                                                         ^^ E501
10 | )
   |
```